lib/pdk/tests/unit.rb

```diff
@@ -31,8 +31,8 @@ def self.rake(task, spinner_text, environment = {})
 
       def self.parallel_with_no_tests?(ran_in_parallel, json_result, result)
         ran_in_parallel && json_result.empty? &&
-          !result[:exit_code].zero? &&
-          result[:stderr].strip =~ %r{Pass files or folders to run$}
+          ((!result[:exit_code].zero? && result[:stderr].strip =~ %r{Pass files or folders to run$}) ||
+           result[:stderr].strip =~ %r{No files for parallel_spec to run against$})
```

Comment:
If this gets much more complicated it may merit some small predicate helper methods to increase legibility

Reply:
Yeah, it's getting a bit nasty isn't it? :)